components/scream/src/physics/p3/p3_functions_f90.cpp

```diff
@@ -1730,26 +1797,11 @@ void cloud_sedimentation_f(
   WorkspaceManager<Spack> wsm(rho_d.extent(0), 4, policy);
   Kokkos::parallel_reduce(policy, KOKKOS_LAMBDA(const MemberType& team, Real& prt_liq_k) {
 
-    uview_1d
-      uqc_incld_d(temp_d[0]),
-      urho_d     (temp_d[1]),
-      uinv_rho_d (temp_d[2]),
-      ulcldm_d   (temp_d[3]),
-      uacn_d     (temp_d[4]),
-      uinv_dzq_d (temp_d[5]),
-      uqc_d      (temp_d[6]),
-      unc_d      (temp_d[7]),
-      unc_incld_d(temp_d[8]),
-      umu_c_d    (temp_d[9]),
-      ulamc_d    (temp_d[10]),
-      uqc_tend_d (temp_d[11]),
-      unc_tend_d (temp_d[12]);
-
     P3F::cloud_sedimentation(
-      uqc_incld_d, urho_d, uinv_rho_d, ulcldm_d, uacn_d, uinv_dzq_d, dnu,
+      qc_incld_d, rho_d, inv_rho_d, lcldm_d, acn_d, inv_dzq_d, dnu,
```

Comment:
This is the implicit conversion you mentioned, right? Is there any downside you can think of to using this? It certainly looks cleaner.

Reply:
> This is the implicit conversion you mentioned, right?

Yes. 

> Is there any downside you can think of to using this?

I don't think there's any downside since the deleted original code is semantically the same. The new code could cease to work if implicit conversions are ever disabled in the View constructors, but I don't know why that would ever happen.